test/cpp/qps/qps-sweep.sh

```diff
@@ -61,37 +61,37 @@ half=`echo $QPS_WORKERS | awk -F, '{print int(NF/2)}'`
 
 for secure in true false; do
```

Comment:
I vote for killing this script and qps_driver entirely before proceeding with this PR. It seems like there would be fever things to fix once we do that.